src/storage/test/IndexTest.cpp

```diff
@@ -93,61 +77,33 @@ TEST(IndexTest, InsertVerticesTest) {
     }
 }
 
-TEST(IndexTest, InsertEdgeTest) {
+TEST(IndexTest, AddEdgeTest) {
```

Comment:
We need to test multiple edges and multiple version edges at here.

Reply:
Support more test case in another PR. This one is big enough.

Reply:
So, you'd better change it with another PR together. 
Because we don't know when that PR is available.Be careful test cannot be covered.